helix-view/src/editor.rs

```diff
@@ -950,6 +961,8 @@ pub struct Editor {
     /// field is set and any old requests are automatically
     /// canceled as a result
     pub completion_request_handle: Option<oneshot::Sender<()>>,
+    pub popup_border: bool,
+    pub menu_border: bool,
```

Comment:
Could these just be helper methods instead of new fields on the `Editor` object?

Reply:
Yes, i changed them